server/subscribe.go

```diff
@@ -269,6 +275,20 @@ func (p *Plugin) addChannelSubscription(newSubscription *ChannelSubscription) er
 	})
 }
 
+func (p *Plugin) isChannelSubscriptionNameUnique(channelId string, subscription *ChannelSubscription) error {
```

Comment:
Suggest changing name to `checkChannelSubscriptionNameUnique` since we are returning an error, rather than a boolean.